tests/rpc_agent_test.py

```diff
@@ -292,29 +319,19 @@ def test_multi_rpc_agent(self) -> None:
 
     def test_mix_rpc_agent_and_local_agent(self) -> None:
         """test to use local and rpc agent simultaneously"""
-        host = "localhost"
-        # use the same port, agents should choose available ports
-        # automatically
-        port1 = 12001
-        port2 = 12001
-        # rpc agent a
         agent_a = DemoRpcAgentAdd(
             name="a",
         ).to_dist(
-            host=host,
-            port=port1,
             lazy_launch=False,
         )
         # local agent b
         agent_b = DemoLocalAgentAdd(
             name="b",
         )
         # rpc agent c
-        agent_c = DemoRpcAgentAdd(
+        agent_c = DemoRpcAgentAdd(  # pylint: disable=E1123
             name="c",
-        ).to_dist(
-            host=host,
-            port=port2,
+            to_dist=True,
```

Comment:
How should we initialize a remote agent in the new setting? Pass a to_dist argument or calling to_dist function?

Reply:
Both methods are okay, see tutorial "Advanced Usage of `to_dist`"